packages/forms/src/index.js

```diff
@@ -27,9 +27,9 @@ const SVG = ({ size = 24, ...props }) =>
   <Box
     as='svg'
     xmlns="http://www.w3.org/2000/svg"
-    width={24}
-    height={24}
-    viewBox="0 0 24 24"
+    width={size}
+    height={size}
+    viewBox={`0 0 ${size} ${size}`}
```

Comment:
Thanks! I think the `viewBox` should remain unchanged here to preserve the aspect ratio of the paths
```suggestion
    viewBox="0 0 24 24"
```